csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -186,18 +207,42 @@ public void RegisterCustomOpLibrary(string libraryPath)
         /// managed by the user (created using the CreateTensorWithDataAsOrtValue API) and it must outlive the session object
         /// to which it is added.
         /// </summary>
-        public void AddInitializer(string name, OrtValue ort_value)
+        public void AddInitializer(string name, OrtValue ortValue)
         {
-            NativeApiStatus.VerifySuccess(NativeMethods.OrtAddInitializer(handle, name, ort_value.Handle));
+            NativeApiStatus.VerifySuccess(NativeMethods.OrtAddInitializer(handle, name, ortValue.Handle));
         }
 
+        /// <summary>
+        /// Set a single session configuration entry as a pair of strings
+        /// If a configuration with same key exists, this will overwrite the configuration with the given config_value
+        /// </summary>
         public void AddSessionConfigEntry(string configKey, string configValue)
```

Comment:
>string configKey, string configValue [](start = 42, length = 36)

String conversion

Reply:
See above - let us perform a purge across the C# codebase in a different PR (this wasn't even added in my PR)